File: FanfictionDB/Data/DAL/nHibernate/Tables/Fanfic.vb

Option Strict Off
Option Explicit On

Imports System
Imports System.Text
Imports System.Collections.Generic


Public Class Fanfic
    Private _id As Integer
    Private _title As String
    Private _author As String
    Private _folder As String
    Private _chapter As String
    Private _count As System.Nullable(Of Integer)
    Private _matchup As String
    Private _crossover As String
    Private _description As String
    Private _internet As String
    Private _storyID As String
    Private _abandoned As System.Nullable(Of Boolean)
    Private _complete As System.Nullable(Of Boolean)
    Private _publish_Date As System.Nullable(Of Date)
    Private _update_Date As System.Nullable(Of Date)
    Private _last_Checked As System.Nullable(Of Date)
    Private _category_Id As Integer

    Public Sub New()
        MyBase.New()
    End Sub
    Public Overridable Property Id() As Integer
        Get
            Return Me._id
        End Get
        Set(value As Integer)
            Me._id = value
        End Set
    End Property
    Public Overridable Property Title() As String
        Get
            Return Me._title
        End Get
        Set(value As String)
            Me._title = value
        End Set
    End Property
    Public Overridable Property Author() As String
        Get
            Return Me._author
        End Get
        Set(value As String)
            Me._author = value
        End Set
    End Property
    Public Overridable Property Folder() As String
        Get
            Return Me._folder
        End Get
        Set(value As String)
            Me._folder = value
        End Set
    End Property
    Public Overridable Property Chapter() As String
        Get
            Return Me._chapter
        End Get
        Set(value As String)
            Me._chapter = value
        End Set
    End Property
    Public Overridable Property Count() As System.Nullable(Of Integer)
        Get
            Return Me._count
        End Get
        Set(value As System.Nullable(Of Integer))
            Me._count = value
        End Set
    End Property
    Public Overridable Property Matchup() As String
        Get
            Return Me._matchup
        End Get
        Set(value As String)
            Me._matchup = value
        End Set
    End Property
    Public Overridable Property Crossover() As String
        Get
            Return Me._crossover
        End Get
        Set(value As String)
            Me._crossover = value
        End Set
    End Property
    Public Overridable Property Description() As String
        Get
            Return Me._description
        End Get
        Set(value As String)
            Me._description = value
        End Set
    End Property
    Public Overridable Property Internet() As String
        Get
            Return Me._internet
        End Get
        Set(value As String)
            Me._internet = value
        End Set
    End Property
    Public Overridable Property StoryID() As String
        Get
            Return Me._storyID
        End Get
        Set(value As String)
            Me._storyID = value
        End Set
    End Property
    Public Overridable Property Abandoned() As System.Nullable(Of Boolean)
        Get
            Return Me._abandoned
        End Get
        Set(value As System.Nullable(Of Boolean))
            Me._abandoned = value
        End Set
    End Property
    Public Overridable Property Complete() As System.Nullable(Of Boolean)
        Get
            Return Me._complete
        End Get
        Set(value As System.Nullable(Of Boolean))
            Me._complete = value
        End Set
    End Property
    Public Overridable Property Publish_Date() As System.Nullable(Of Date)
        Get
            Return Me._publish_Date
        End Get
        Set(value As System.Nullable(Of Date))
            Me._publish_Date = value
        End Set
    End Property
    Public Overridable Property Update_Date() As System.Nullable(Of Date)
        Get
            Return Me._update_Date
        End Get
        Set(value As System.Nullable(Of Date))
            Me._update_Date = value
        End Set
    End Property
    Public Overridable Property Last_Checked() As System.Nullable(Of Date)
        Get
            Return Me._last_Checked
        End Get
        Set(value As System.Nullable(Of Date))
            Me._last_Checked = value
        End Set
    End Property
    Public Overridable Property Category_Id() As Integer
        Get
            Return Me._category_Id
        End Get
        Set(value As Integer)
            Me._category_Id = value
        End Set
    End Property
End Class

File: Downloader/HtmlScraper/Data/DAL/nHibernate/Tables/Fanfic.cs

namespace HtmlScraper.Data.DAL.nHibernate.Tables
{


    public class Fanfic
    {
        private int _id;
        private string _title;
        private string _author;
        private string _folder;
        private string _chapter;
        private int? _count;
        private string _matchup;
        private string _crossover;
        private string _description;
        private string _internet;
        private string _storyID;
        private bool? _abandoned;
        private bool? _complete;
        private DateTime? _publish_Date;
        private DateTime? _update_Date;
        private DateTime? _last_Checked;
        private int _category_Id;

        public Fanfic() : base()
        {
        }
        public virtual int Id
        {
            get
            {
                return _id;
            }
            set
            {
                _id = value;
            }
        }
        public virtual string Title
        {
            get
            {
                return _title;
            }
            set
            {
                _title = value;
            }
        }
        public virtual string Author
        {
            get
            {
                return _author;
            }
            set
            {
                _author = value;
            }
        }
        public virtual string Folder
        {
            get
            {
                return _folder;
            }
            set
            {
                _folder = value;
            }
        }
        public virtual string Chapter
        {
            get
            {
                return _chapter;
            }
            set
            {
                _chapter = value;
            }
        }
        public virtual int? Count
        {
            get
            {
                return _count;
            }
            set
            {
                _count = value;
            }
        }
        public virtual string Matchup
        {
            get
            {
                return _matchup;
            }
            set
            {
                _matchup = value;
            }
        }
        public virtual string Crossover
        {
            get
            {
                return _crossover;
            }
            set
            {
                _crossover = value;
            }
        }
        public virtual string Description
        {
            get
            {
                return _description;
            }
            set
            {
                _description = value;
            }
        }
        public virtual string Internet
        {
            get
            {
                return _internet;
            }
            set
            {
                _internet = value;
            }
        }
        public virtual string StoryID
        {
            get
            {
                return _storyID;
            }
            set
            {
                _storyID = value;
            }
        }
        public virtual bool? Abandoned
        {
            get
            {
                return _abandoned;
            }
            set
            {
                _abandoned = value;
            }
        }
        public virtual bool? Complete
        {
            get
            {
                return _complete;
            }
            set
            {
                _complete = value;
            }
        }
        public virtual DateTime? Publish_Date
        {
            get
            {
                return _publish_Date;
            }
            set
            {
                _publish_Date = value;
            }
        }
        public virtual DateTime? Update_Date
        {
            get
            {
                return _update_Date;
            }
            set
            {
                _update_Date = value;
            }
        }
        public virtual DateTime? Last_Checked
        {
            get
            {
                return _last_Checked;
            }
            set
            {
                _last_Checked = value;
            }
        }
        public virtual int Category_Id
        {
            get
            {
                return _category_Id;
            }
            set
            {
                _category_Id = value;
            }
        }
    }
}